docs/multi-stage-query/reference.md

```diff
@@ -501,10 +506,11 @@ When using the sort-merge algorithm, keep the following in mind:
 
 - All join types are supported with `sortMerge`: LEFT, RIGHT, INNER, FULL, and CROSS.
 
-The following example  runs using a single sort-merge join stage that receives `eventstream`
-(partitioned on `user_id`) and `users` (partitioned on `id`) as inputs. There is no limit on the size of either input.
+The following example shows a single sort-merge join stage where it explicitly set `sqlJoinAlgorithm` to `sortMerge` using the SET command. This query also takes `eventstream` (partitioned on `user_id`) and `users` (partitioned on id) as `inputs`, with no limit on the size of either input.
```

Comment:
```suggestion
The following query runs a single sort-merge join stage that takes the following inputs:
* `eventstream` partitioned on `user_id`
* `users` partitioned on `id`

There is no limit on the size of either input.
The SET clause sets the `sqlJoinAlgorithm` context parameter so that Druid applies the sort-merge join algorithm for the query.
```